nixos/modules/security/pam.nix

```diff
@@ -212,6 +212,15 @@ let
         '';
       };
 
+      enableKwallet = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Enable pam_kwallet to automatically unlock user KDE Wallets
+          upon login (if their wallet password matches their login password)
```

Comment:
Your wallet just won't be auto-unlocked, and KDE will prompt you for the wallet password when it is needed.  I'm I will make sure to test things like that prior to merging this change.